initdb.py

```diff
@@ -1,56 +1,87 @@
 # Initializes a MySQL Database
 # 1) connect to server
 # 2) create database with cursor
-# 3) connect to database
+# 3) connect to created database
 # 4) create tables with cursor
-# 5) commit transactoins
 # 6) exit
```

Comment:
1) Now that (5) is gone, the numbering is messed up
2) I'm not really sure this is even needed.

@ASAAR thoughts?